lib/routes/searches.js

```diff
@@ -3,12 +3,18 @@ const Search = require('../model/Search');
 
 router.post('/', (request, response) => {
   try {
-    const tournament = request.body.tournament;
     const user = request.body.user;
+    const tournament = request.body.tournament;
+    const discipline = request.body.discipline;
+    const position = request.body.position;
+    const message = request.body.message;
```

Comment:
```suggestion
    const user = request.body.user;
    const tournament = request.body.tournament;
    const discipline = request.body.discipline;
    const position = request.body.position;
    const message = request.body.message;
```
```
const{user,tournament,discipline,position,message} = request.body;
```
You could reduce the code a little bit like this 😄